# Gross Income And Exclusions

**Selected Examples**

McGraw Hill's Taxation of Individuals and Business Entities 2025

Spilker, Ayers, Lewis, Weaver, Barrick, Robinson, Worsham

In [1]:
import os
import sys

mod_path = os.path.abspath(os.path.join(".."))
if mod_path not in sys.path:
    sys.path.append(mod_path)

from income_tax import TaxableSocialSecurity

---

**Example 5-1**

In April, Gram used part of the life insurance proceeds she received from Gramps’s death to purchase 50 shares in Acme Corporation for 30 per share. From April to the end of December, the value of the shares fluctuated between 40 and 25, but on December 31, the shares were worth 35. If Gram does not sell the shares, how much income from her stockholdings in Acme Corporation does she realize for the year?

In [7]:
beginning_value = 50 * 30
ending_value = 50 * 35
realized = ending_value - beginning_value

print(f"""
Investment in Acme at April 30:      $ {beginning_value:,.0f}
Investment in Acme at December 31:   $ {ending_value:,.0f}
                                     -------
Gain Realized:                       $   {realized:,.0f}
Gain Recognized:                     $     {0:,.0f}
""")


Investment in Acme at April 30:      $ 1,500
Investment in Acme at December 31:   $ 1,750
                                     -------
Gain Realized:                       $   250
Gain Recognized:                     $     0



**Answer:** 0. Unless Gram sells the stock, she does not enter into a transaction resulting in a measurable change of property rights with a second party. Thus, she does not realize income even though she experienced an economic benefit from the appreciation of the stock from 30 per share to 35 per share.

---

**Example 5-2**

Suppose during March, Gram paid no rent to her neighbor (also her landlord). Although the neighbor typically charges 750 per month for rent, he allowed Gram to live rent-free in exchange for babysitting his infant son. What income would Gram and Gram’s neighbor realize and recognize on this exchange?

**Answer**: 750. Gram and the neighbor each would recognize 750 of income for March. The neighbor recognizes 750 of rental receipts because this is the value of the babysitting services the neighbor received in lieu of a cash payment for rent from Gram (an economic benefit the neighbor realized through the exchange). Gram recognizes 750 of babysitting income because this is the value of the services provided to her neighbor (an economic benefit was realized because Gram was not required to pay rent).

---

**Example 5-3**

In 2023 Courtney paid 3,500 in Ohio state income taxes, and she included this payment with her other itemized deductions when she filed her federal income tax return in March of 2024. Courtney filed her 2023 federal return as a head of household and claimed 22,900 of itemized deductions (including 3,500 state income taxes, 2,500 of real estate taxes, 8,000 of mortgage interest expense, and 8,900 of charitable contributions). She also filed an Ohio state income tax return in March of 2024 and discovered she only owed 3,080 in Ohio income tax for 2023. Hence, Courtney received a 420 refund of her Ohio income tax in June of 2024.

Applying the tax benefit rule, how much of the 420 refund, if any, is Courtney required to include in her gross income for 2024 given that the standard deduction for head of household filing status in 2023 was 20,800?

**Answer:** All 420. Courtney is required to include the entire refund in her 2024 gross income because her itemized deduction for the 420 of state income taxes that she overpaid last year reduced her taxable income by 420. Accordingly, because she received a tax benefit (deduction) for the entire 420 overpayment, she must include it all in gross income for 2024. See the following for the calculation of the amount of tax benefit Courtney received from the 420 overpayment of taxes in 2023.

| Deduction | Amount | Explanation |
| --- | :---: | --- |
| (1) Itemized deductions | 22,900 | |
| (2) 2023 standard deduction | 20,800 | Head of household filing status |
| (3) Reduction in taxable income | 22,900 | Greater of (1) and (2) |
| (4) Itemized deductions adjusted for 420 refund | 22,480 | (1) - 420 |
| (5) Reduction in taxable income after adjustment | 22,480 | Greater of (2) and (4) |
| **Taxable benefit due to prior deduction of 420 refund** | **420** | (3) - (5) |

**What if:** Let’s consider alternative fact patterns provided in Scenarios A and B to further illustrate the application of the tax benefit rule.

How much of the 420 refund would Courtney include in her 2024 gross income in Scenarios A and B?

- **Scenario A:** In 2023 Courtney’s itemized deductions, including 3,500 in state taxes, were 15,500.
    - **Answer Scenario A:** 0. As computed below, Courtney received 0 tax benefit from the $420 tax overpayment in 2023, so she need not include any of the refund in her 2024 gross income.

- **Scenario B:** In 2023 Courtney’s itemized deductions, including 3,500 in state taxes, were 21,150.
    - **Answer Scenario B:** 350. As computed below, Courtney received a $350 tax benefit (reduction in taxable income) from the 420 state tax overpayment in 2023, so she must include 350 of the refund in her 2024 gross income.

| Deduction | Scenario A | Scenario B | Explanation
| --- | :---: | :---: | --- |
| (1) Itemized deductions | 15,500 | 21,500 | |
| (2) 2023 standard deduction | 20,800 | 20,800 | Head of household filing status |
| (3) Reduction in taxable income | 20,800 | 21,500 | Greater of (1) and (2) |
| (4) Adjusted itemized deductions | 15,080 | 20,730 | (1) - 420 |
| (5) Reduction in taxable income adjusted for 420 | 20,800 | 20,800 | Greater of (2) and (4) |
| **Tax benefit due to prior deduction** | **0** | **350** | (3) - (5) |

- **What if:** In 2023 Courtney’s itemized deductions were 21,150 and included 10,000 of state taxes (10,500 total state taxes, limited by tax law to 10,000). We discuss the 10,000 state and local tax limitation in the Individual Deductions chapter. How much of the 420 refund would Courtney include in her 2024 gross income?
     - **Answer:** 0. Courtney received 0 tax benefit from the 420 tax overpayment in 2023. Her itemized deduction for state taxes adjusted for the 420 refund would still be 10,000 (10,500 − 420 = 10,080, limited to 10,000) and her total itemized deductions would still be 21,150. So, she need not include any of the refund in her 2024 gross income.



---

**Example 5-4**

Courtney is a cash-method taxpayer. Based on her outstanding performance at work, Courtney earned a 4,800 year-end bonus. On December 28, Courtney’s supervisor told her that her bonus was issued as a separate check and that Courtney could pick up the check in the accounting office anytime. Courtney did not pick up the check until January 2 of the next year, and she did not cash it until late January. When does Courtney realize and recognize the 4,800 income?

**Answer:** On December 28 of the tax year in question. Courtney constructively received the check that year because it was unconditionally available to her on December 28, she was aware of the check’s availability, and there were no restrictions on her control over the check. Courtney must include the 4,800 bonus in gross income for that year, even though she did not actually receive the funds until late January of the next year.


---

**Example 5-5**

**What if:** Courtney would like to begin saving for Deron’s college tuition. If Courtney were to direct EWD to deposit part of her salary in Deron’s bank account, who would owe tax on the salary income?

**Answer:** Courtney would be taxed on her entire salary as income because she earned the income. The payment to Deron would be treated as a gift and would not be taxable to him (gifts are excluded from the recipient’s income, as discussed later in the chapter).

**What if:** Suppose Courtney wanted to shift her rental income to Deron. What would she need to do to ensure that her rental income is taxed to Deron?

**Answer:** Courtney would have to transfer her ownership in the rental property to Deron in order for Deron to be taxed on the rental income.

---

**Example 5-6**

In the year prior to their divorce, Courtney and Al Wilson lived in Ohio, a common law state, and filed married filing separately. That year, Al earned 90,000 of annual salary and Courtney earned 70,000. How much of the income earned by Al and by Courtney in the year prior to the divorce did Al report on his individual tax return? How much income did Courtney report on her individual tax return?

**Answer:** Because they resided in a common law state, Al reports the 90,000 that he earned on his own tax return, and Courtney reports the entire 70,000 she earned on her own tax return.

**What if:** How much of this income would Al have been required to include on his individual tax return, and how much of this income would Courtney have been required to report on her individual tax return, if they lived in California, a community property state?

**Answer:** If they resided in a community property state, both Al and Courtney would have included 80,000 of the 160,000 the couple had jointly earned [(Al’s 90,000 + Courtney’s 70,000)/2] on their respective individual tax returns.


---

**Example 5-7**

EWD pays Courtney a salary of 158,000. In addition, Courtney earned and received 19,500 in fees from consulting work she did on weekends independent of her employment with EWD. She incurred 1,500 in miscellaneous expenses for supplies and transportation while doing the consulting work. What is Courtney’s total income from services (earned income) from her employment and from her self-employment activities?

**Answer:** 176,000, consisting of her 158,000 salary and her 18,000 business income from her consulting activities (19,500 revenue less 1,500 expenses related to her consulting activities). Note that business deductions are for AGI deductions that taxpayers subtract from their gross business income to derive net business income or loss, reported on Schedule 1 (line 3) of Form 1040

---

**Example 5-8**

Courtney owns 1,000 shares of Home Depot stock that she registered in a dividend reinvestment plan. Under this plan, all dividends are automatically used to purchase more shares of the stock. This year, Home Depot declared and paid 700 in dividends on Courtney’s stock. Must Courtney include the dividends in her gross income for the year?

**Answer:** Yes. Courtney includes the 700 of dividends (unearned income) in her gross income. The fact that Courtney chose to reinvest the dividends does not affect their taxability because she received an economic benefit and change in property rights associated with the dividends.

---

**Example 5-9**

Gram purchased a 100,000, three-year certificate of deposit (CD) with a portion of the life insurance proceeds she received on Gramps’s death. At year-end, her CD account is credited with 4,100 of interest, and her savings account is credited with 650 of interest. Courtney had a total of 271 of interest credited to her savings account and 50 credited to her checking account during the year. How much interest must Gram and Courtney include in their gross income for the year?

**Answer:**  **4,750(4,100 + 650)** for Gram and **321(271 + 50)** for Courtney. Gram must include the 4,100 of interest credited to her CD account and the 650 of interest credited to her savings account, regardless of whether she withdraws the interest or not. Courtney must include the 271 of interest credited to her savings account and the 50 credited to her checking account, regardless of whether she withdraws the interest or not.

---

**Example 5-10**

Courtney owns a condo in town that she rents to tenants. This year, the condo generated 14,000 of rental revenue. Courtney incurred 4,000 in real estate taxes, 2,500 in utility expenses, 500 in advertising expenses, and 2,000 of depreciation and other expenses associated with the rental. What effect does the rental have on Courtney’s gross income? What effect does the rental have on Courtney’s taxable income?

**Answer:** The rent increases Courtney’s gross income by 14,000. However, after considering her allowable deductions for the rental, Courtney will report only 5,000 of taxable income from rental activities, computed as follows:

| Description | Amount |
| --- | :---: |
| Rental revenue | 14,000 |
| Less allowable deductions: |
| Real estate taxes | (4,000) |
| Utilities | (2,500) |
| Advertising | (500) |
| Depreciation and other expenses | (2,000) |
| Total rental expenses | (9,000) |
| **Net (taxable) rental income** | **5,000** |

Under certain circumstances, the rental income will also generate a “deduction for qualified business income,” which would reduce Courtney’s taxable income.

---

**Example 5-11**

In January of this year, Gram purchased an annuity from UBET Insurance Co. that will pay her 10,000 per year for the next 15 years. Gram received the first 10,000 payment in December. Gram paid 99,000 for the annuity and will receive 150,000 over the life of the annuity (15 years × 10,000 per year). How much of the 10,000 payment received by Gram in December should she include in her gross income?

**Answer:** $3,400. Because Gram purchased a fixed-payment annuity, the portion of the annuity payment included in gross income is calculated as follows:

| Description | Amount | Explanation |
| --- | :---: | --- |
| (1) Investment in annuity contract | 99,000 | |
| (2) Number of payment | 15 | |
| (3) Return of capital per payment | 6,600 | (1) / (2) |
| (4) Amount of each payment: | 10,000 | |
| **Gross income per payment** | **3,400** | (4) - (3) |

**What if:** Assume the annuity Gram purchased pays her 1,000 per month over the remainder of her life. Gram (70 years old) paid 99,000 for the annuity, and she received her first 1,000 payment in December of this year. How much income would she recognize on the 1,000 payment?

**Answer:** $484.40, computed as follows:

| Description | Amount | Explanation |
| --- | :---: | --- |
| (1) Investment in annuity contract | 99,000 | |
| (2) Expected return multiple | 16 | Exibit 5-1, 70 years old |
| (3) Amount of each payment | 1,000 | |
| (4) Expected return | 192,000 | (2) x (3) x 12 months |
| (5) Return of capital percentage | 51.56% | (1) / (4) |
| (6) Return of capital per payment | 515.60 | (3) x (5) |
| **Taxable income per payment** | **484.40** | (3) - (6) |

After Gram receives her entire 99,000 as a return of capital, each subsequent 1,000 annuity payment would be fully taxable.

---

**Example 5-12**

On December 31, Gram sold her 50 shares of Acme Corporation stock for 40 per share. Gram also paid 150 in broker’s commissions on the sale. Gram originally purchased the shares in April of this year for 30 per share. How much gross income does Gram recognize from the stock sale?

**Answer:** Gram recognizes 350 gross income on the sale, computed as follows:

| Description | Amount |
| --- | :---: |
| Sales proceeds (40 x 50 shares) | 2,000 |
| Less: Selling expenses | -150 |
| = Amount realized | 1,850 |
| Less: Tax basis (investment) in property sold (30 x 50 shares) | -1,500 |
| **Gain (Loss) on sale** | **350** |

---

**Example 5-13**

**What if:** Suppose that Courtney is a 40 percent partner in KJZ partnership. KJZ reported 20,000 of business income and 3,000 of interest income for the year. KJZ also distributed 1,000 of cash to Courtney. What amount of gross income from her ownership in KJZ partnership would Courtney report for the current year assuming her tax basis in KJZ is 15,000 before considering the distribution?

**Answer:** 9,200, consisting of 8,000 of business income (20,000 × 40%) and 1,200 of interest income (3,000 × 40%). Courtney would not include the 1,000 distribution in her gross income because it was a return of her investment.

---

**Example 5-14**

In addition to paying child support, under the divorce decree executed in 2023, Al is required to pay Courtney 20,000 cash each year until Courtney or Al dies. The decree does not designate this amount as a payment for something other than alimony, and Al and Courtney do not live together. Does this payment qualify as alimony?

**Answer:** Yes. These payments qualify as alimony for tax purposes because (1) they are cash payments made under a divorce decree, (2) they are not designated as something other than alimony, (3) Al and Courtney do not live together, and (4) the payments cease on Courtney’s death. In the current year, Al made a 20,000 alimony payment to Courtney. Because their divorce agreement was executed in 2023, Courtney does not include the 20,000 in her gross income, and Al does not treat the payment as a deduction for AGI.

**What if:** Suppose that Courtney and Al’s divorce agreement was executed in 2018. What amount of the 20,000 payment would Courtney include in her gross income and what amount would Al deduct?

**Answer:** Because the divorce agreement was executed before January 1, 2019, Courtney would include the 20,000 alimony payment in gross income, and Al would treat the payment as a deduction for AGI on Schedule 1 (line 19) of Form 1040.

---

**Example 5-15**

As required by the divorce decree, Al made 10,000 in child support payments during the year to Courtney to help her support Deron. Al is required to make child support payments until Deron is 18 years old. Is the current-year payment included in Courtney’s gross income and deductible by Al?

**Answer:** No. The 10,000 of child support payments is not income to Courtney, and Al does not deduct the payments. Child support payments are not includable in gross income by the recipient, and they are not deductible by the payor.

---

**Example 5-16**

As part of the divorce agreement executed in 2023, Al transferred his interest in their joint residence to Courtney. The couple did not have any outstanding debt on the home. At the time of the divorce, the home was valued at 500,000 and Al’s share was valued at 250,000. Does the home ownership transfer to Courtney generate taxable income for her and a deduction for Al?

**Answer:** Property divisions upon divorce are not taxable events. Consequently, Courtney would not have recognized the 250,000 as gross income, and Al would not have deducted the 250,000 transfer.

---

**Example 5-17**

After devoting much of her free time during the year to filling out sweepstakes entries, Gram hit the jackpot. She won a WaveRunner worth 9,500 in a sweepstakes sponsored by Reader’s Digest. How much of the prize, if any, must Gram include in her gross income?

**Answer:** Gram must include the full 9,500 value of the WaveRunner in her gross income. Note that because she must pay taxes on the winnings, she is not really getting the WaveRunner for “free.”

---

**Example 5-18**

Gram received $7,200 of Social Security benefits this year. Suppose that Gram’s modified AGI is $16,000. What amount of the Social Security benefits must Gram include in her gross income?

In [2]:
social_security = 7200
modified_agi = 16000
taxable_ss = TaxableSocialSecurity(modified_agi, social_security).single()
print(f"Answer: ${taxable_ss:,.0f}")

Answer: $0


**Answer:** 0. Gram’s modified AGI plus one-half of her Social Security benefits is 19,600 [16,000 modified AGI + (7,200 Social Security benefits × 50%)], which is below 25,000. Hence, Gram may exclude the entire 7,200 of Social Security income from gross income.

**What if:** Assume that Gram received 7,200 of Social Security benefits this year and that her modified AGI is 26,400. What amount of benefits must Gram include in her gross income?

In [3]:
social_security = 7200
modified_agi = 26400
taxable_ss = TaxableSocialSecurity(modified_agi, social_security).single()
print(f"Answer: ${taxable_ss:,.0f}")

Answer: $2,500


**Answer:** 2,500. Gram is single and her modified AGI + 50 percent of Social Security benefits falls between 25,000 and 34,000. Her taxable Social Security benefits are the lesser of (a) 50 percent of the Social Security benefits (7,200 × 50% = 3,600) or (b) 50 percent of [26,400 modified AGI + 3,600 (50 percent of Social Security benefits) − 25,000] = 2,500. Thus, her taxable Social Security benefits are 2,500.

**What if:** Assume that Gram received 7,200 of Social Security benefits this year and that her modified AGI is 31,400. What amount of benefits must Gram include in her gross income?

In [4]:
social_security = 7200
modified_agi = 31400
taxable_ss = TaxableSocialSecurity(modified_agi, social_security).single()
print(f"Answer: ${taxable_ss:,.0f}")

Answer: $4,450


**Answer:** 4,450. Gram is single and her modified AGI + 50 percent of Social Security benefits exceeds 34,000. Her taxable Social Security benefits are the lesser of (a) 85 percent of Social Security benefits (7,200 × 85% = 6,120) or (b) 85 percent of [31,400 modified AGI + 3,600 (50 percent of Social Security benefits) − 34,000] = 850, plus the lesser of (i) 4,500 or (ii) 50 percent of Social Security benefits (50% × 7,200 = 3,600). This calculation simplifies to the lesser of (a) 6,120 or (b) 4,450 [850 + lesser of (i) 4,500 or (ii) 3,600]. Thus, Gram’s taxable Social Security benefits are 4,450.

**What if:** Assume that Gram received 7,200 of Social Security benefits this year and that her modified AGI is 50,000. What amount of benefits must Gram include in her gross income?

In [5]:
social_security = 7200
modified_agi = 50000
taxable_ss = TaxableSocialSecurity(modified_agi, social_security).single()
print(f"Answer: ${taxable_ss:,.0f}")

Answer: $6,120


**Answer:** $6,120($7,200 × 85%).

---

**Example 5-19**

To create more space for Gram, Courtney received EWD architectural design services as part of her compensation package and at a substantial discount. EWD’s services were valued at 35,000, but Courtney was charged only 22,000. How much of the discount must Courtney include in gross income?

**Answer:** 6,000, computed as follows:

| Description | Amount | Explanation |
| --- | :---: | --- |
| (1) Value of services EWD provided to Courtney | 35,000 | |
| (2) Courtney's cost of the services | 22,000 | |
| (3) Discount on services | 13,000 | (1) - (2) |
| (4) Excludable discount | 7,000 | (1) x 20% |
| **Discount in excess of 20% is included in gross income | **6,000** | (3) - (4) |

---

**Example 5-20**

At the beginning of January, EWD provides Courtney with a 100,000 zero-interest loan. Assume the applicable federal interest rate (compounded semiannually) is 4 percent. Courtney used the loan proceeds to acquire several personal-use assets (an automobile and other items). What amount is Courtney required to include in gross income in the current year?

**Answer:** $4,000, computed as follows:

| Description | Amount | Explanation |
| --- | :---: | --- |
| (1) Loan principal | 100,000 | |
| (2) Applicable federal interest rate compounded semiannually | 4% | |
| (3) Interest on loan principal at federal rate | 4,000 | (1) x (2) |
| (4) Interest paid by Courney | 0 | |
| **Imputed interest included in Courtney's gross income** | **4,000** | (3) - (4) |

**What if:** Assume the same facts except that EWD had loaned Courtney 10,000 at a zero-interest rate. How much imputed interest income would Courtney be required to include in her gross income for the year?

**Answer:** 0. The imputed interest rules don’t apply because the loan did not exceed 10,000.

---

**Example 5-21**

In the previous example, Courtney borrowed 100,000 from EWD. Because EWD wants to keep Courtney as an employee, it agreed to forgive 10,000 of loan principal at the end of each year that Courtney stays on board. On December 31 of this year, EWD formally cancels 10,000 of Courtney’s indebtedness. How much of this debt relief is Courtney required to include in gross income this year?

**Answer:** All 10,000 is included in Courtney’s gross income.

---

**Example 5-22**

Courtney holds a 10,000 City of Cincinnati municipal bond. The bond pays 5 percent interest annually. Courtney acquired the bond a few years ago. The city used the proceeds from the bond issuance to help pay for renovations on a major league baseball stadium. In late December, Courtney received 500 of interest income from the bond for the year. How much of the 500 interest from the municipal bond may Courtney exclude from her gross income?

**Answer:** All $500 because the interest is from a municipal bond.


---

**Example 5-23**

**What if:** Assume that in October of this year, Courtney sold her home in Cincinnati. Courtney and her ex-husband purchased the home four years ago for 400,000, and Courtney received the house in the divorce settlement and lived there until she moved to Kansas City in January. She sold the home for 550,000. How much gain may she exclude on the sale of the home?

**Answer:** All 150,000. Because Courtney satisfies the two-year ownership and two-year use tests, she may exclude up to 250,000 of gain from the sale of her home. Thus, Courtney may exclude the entire 150,000 gain that she realized on the sale (550,000 sales price less 400,000 basis).

**What if:** Assume the same facts except Courtney sold the home for 700,000. How much gain may she exclude on the sale of the home?

**Answer:** 250,000. Because Courtney satisfies the two-year ownership and two-year use tests, she may exclude 250,000 of the 300,000 gain from the sale of her home (700,000 sales price less 400,000 basis). Thus, Courtney excludes 250,000 of the gain and recognizes the remaining $50,000 taxable gain on the sale.




---

**Example 5-24**

EWD paid 6,000 this year for Courtney’s health insurance premiums and 150 in premiums for her 40,000 group-term life insurance policy. How much of the 6,150 in benefits can Courtney exclude from her gross income?

**Answer:** Courtney can exclude all 6,150 in benefits from her gross income. All health insurance premiums paid by an employer on an employee’s behalf are excluded from the employee’s income. In addition, premiums employers pay on an employee’s behalf for group-term life insurance (up to 50,000 of coverage) are also excluded from the employee’s gross income.

---

**Example 5-25**

In December, Courtney mailed a newsletter to several dozen friends and relatives with a recent picture of her son, daughter, and mother. Courtney printed both the newsletter and the photos on printers at work (with permission of EWD). Courtney would have paid 55 for the duplicate newsletters and photos at a nearby copy center. How much of this 55 benefit that Courtney received from EWD may she exclude from her gross income?

**Answer:** All 55 is excluded. The 55 benefit is considered a nontaxable de minimis (so minor as to merit disregard) fringe benefit because it is small in amount and infrequent.

---

**Example 5-26**

Ellen, Courtney’s daughter, received a 5,700 scholarship from the University of Missouri–Kansas City that pays 5,400 of her tuition and provides 300 cash for books. Ellen spent 350 on books. How much of the scholarship may she exclude from gross income?

**Answer:** All 5,700 is excluded. Ellen may exclude all of the scholarship for tuition, and she may exclude the 300 cash she received for books because she spent all 300 purchasing her books for school.

**What if:** How much is Ellen allowed to exclude if she spent only 250 on books?

**Answer:** 5,650 is excluded. Ellen excludes all of the scholarship for her tuition and 250 of the 300 in cash payments because she spent 250 on books. She must include the excess cash payment of 50(300 − 250) in her gross income.





---

**Example 5-27**

Ellen graduated from high school last year. As a graduation present, Gram purchased a 1,500 travel package for Ellen so that Ellen could go on a Caribbean cruise. Last year, Ellen also received an inheritance of 2,000 from Gramps’s estate. How much of the 1,500 graduation gift from Gram and the 2,000 inheritance from Gramps is Ellen allowed to exclude from her gross income?

**Answer:** All of it. Ellen is allowed to exclude the entire gift and the entire amount of the inheritance from her gross income. Consequently, she does not recognize any gross income from these transactions.

---

**Example 5-28**

Gramps received 200,000 of accelerated death benefits from a life insurance policy last year when he was diagnosed with terminal cancer with a life expectancy of less than one year. How much of the 200,000 accelerated death benefits is Gramps allowed to exclude from gross income?

**Answer:** All of it because Gramps was medically certified as terminally ill with an illness expected to cause death within 24 months.

**What if:** Due to financial issues, several years ago Gramps transferred a 100,000 life insurance policy on his life to a business associate for 5,000. Since that time, the business associate has continued to pay the annual premiums on the policy (totaling 20,000 before Gramps’s death). Upon Gramps’s death, the life insurance company paid the business associate the policy’s 100,000 face value. How much of the 100,000 insurance payment is the business associate allowed to exclude from gross income?

**Answer:** 25,000. Because the life insurance policy was purchased from Gramps for valuable consideration, the business associate may exclude the 100,000 proceeds up to the sum of the purchase price of the policy (5,000) and any subsequent premiums (20,000) for a total of 25,000, with the remaining proceeds (100,000 − 5,000 − 20,000 = 75,000) taxable as ordinary income.


---

**Example 5-29**

Assume Courtney is considering a transfer to EWD’s overseas affiliate. If she transfers, she anticipates she will earn approximately 140,000 in salary. How much of her expected 140,000 annual salary will Courtney be allowed to exclude from her gross income, assuming she meets the residency requirements?

In [6]:
foreign_income = 140000
maximum_exclusion = 126500       # 2024 Inflation Indexed amount
exclusion = min(foreign_income, maximum_exclusion)
print(f"Answer: ${exclusion:,.0f}")

Answer: $126,500



**Answer:** She is eligible to exclude 126,500 of her 140,000 of compensation from U.S. taxation in 2024. However, her entire 140,000 salary may be subject to the foreign country’s income tax.


**What if:** Assume Courtney decides to transfer, but she expects her 12-month qualifying period to include only 200 days in the first year and 140 days in the second year. How much of her expected 77,000 salary in the first year will she be allowed to exclude from gross income? Assume that there are 365 days in the year.

In [7]:
expected_salary = 77000
first_year_days = 200
days_in_year = 365
exclusion = maximum_exclusion * (first_year_days / days_in_year)
first_year_exclusion = min(expected_salary, exclusion)
print(f"Answer: ${first_year_exclusion:,.0f}")

Answer: $69,315



**Answer:** 69,315 [126,500 full exclusion × 200/365 (days in foreign country/days in year)].

**What if:** Assume that Courtney expects her 12-month qualifying period to include 200 days in the first year and that she expects her salary to be 45,000 during this period. How much of her expected 45,000 salary will she be allowed to exclude from gross income?

In [9]:
expected_salary = 45000
first_year_exclusion = min(expected_salary, exclusion)
print(f"Answer: ${first_year_exclusion:,.0f}")

Answer: $45,000


**Answer:** All 45,000. She is eligible to exclude up to 69,315 of salary from income (see above computation).

**What if:** Assume that if Courtney transfers, EWD’s overseas affiliate will also pay for her housing while overseas (25,000 per year). How much of the 25,000 housingnpayments may Courtney exclude assuming she lives in the country for the entire year?

In [10]:
housing = 25000
limit_16_pct = maximum_exclusion * .16
limit_14_pct = maximum_exclusion * .14
excess = housing - limit_16_pct
housing_exclusion = min(excess, limit_14_pct)
print(f"Answer: ${housing_exclusion:,.0f}")


Answer: $4,760


**Answer:** 4,760. Because Courtney meets the requirements for the foreign-earned income exclusion, she may exclude the employer-provided housing costs that exceed 20,240(16% × 126,500), up to a maximum exclusion of 17,710(14% × 126,500). Thus, Courtney may exclude 4,760 [the lesser of (a) 4,760 (25,000 housing cost less 20,240) or (b) 17,710].



---

**Example 5-30**

In February, Courtney’s cousin, Kelsey, was struck and injured by a bus while walking in a crosswalk. Because the bus driver was negligent, the bus company settled Kelsey’s claim by paying her 11,500 for medical expenses and 23,500 for emotional distress associated with the accident. How much of the 35,000 Kelsey received from the bus company may she exclude from her gross income?

**Answer:** All 35,000. Kelsey may exclude the 11,500 she received for medical expenses and the 23,500 payment she received for emotional distress because these damages were associated with Kelsey’s physical injury.

**What if:** Assume Kelsey sued the bus company and was awarded 50,000 in punitive damages. How much of the 50,000 would Kelsey be able to exclude from her gross income?

**Answer:** 0. Payments for punitive damages are not excludable. Kelsey would be required to include the entire 50,000 in her gross income.



---

**Example 5-31**

Courtney purchased disability insurance last year. In late April of this year, she broke her wrist in a mountain biking accident and could not work for two weeks. Courtney’s doctor bills totaled 2,000, of which 1,600 was reimbursed by her health insurance purchased by EWD. How much of the 1,600 health insurance reimbursement for medical expenses is Courtney allowed to exclude from her gross income?

**Answer:** All 1,600 is excluded. All medical expense reimbursements from health insurance are excluded from a taxpayer’s gross income.

Courtney also received 6,000 for lost wages due to the accident from her disability insurance policy. How much of the 6,000 is Courtney allowed to exclude from her gross income?

**Answer:** All 6,000 is excluded. Courtney can exclude the entire amount because she paid the premiums on the policy.

**What if**: How much of the 6,000 payment for lost wages from the disability insurance policy would Courtney exclude if EWD paid the disability insurance premium on her behalf as a nontaxable fringe benefit?

**Answer:** 0 would be excluded. Because the disability policy premiums paid by EWD were treated as nontaxable fringe benefits, Courtney would include the payment in gross income and be taxed on the entire 6,000.

**What if:** How much of the 6,000 payment for lost wages from the disability insurance policy would Courtney be allowed to exclude if she paid half the cost of the policy personally and EWD paid the other half as a taxable fringe benefit?

**Answer:** All 6,000 would be excluded. Disability benefits received from policies purchased by an individual are excludable. Likewise, disability benefits received from policies purchased by an employer as a taxable fringe benefit are excludable. Thus, the entire 6,000 payment would be excludable. Note that if EWD had paid for half the cost of the policy as a nontaxable fringe benefit, Courtney would have been able to exclude 3,000, not 6,000.

---